boards/ti/sk_am62/doc/index.rst

```diff
@@ -59,15 +69,19 @@ SD Card
 
 Download TI's official `WIC`_ and flash the WIC file with an etching software
 onto an SD-card. This will boot Linux on the A53 application cores of the EVM.
-These cores will then load the zephyr binary on the M4 core using remoteproc.
+While programming for the M4 core, the A53 cores will then load the zephyr binary on the M4 core using remoteproc.
 
 The default configuration can be found in
+- M4F Core
 :zephyr_file:`boards/ti/sk_am62/sk_am62_am6234_m4_defconfig`
+- A53 Core
+:zephyr_file:`boards/ti/sk_am62/sk_am62_am6234_a53_defconfig`
 
```

Comment:
Just drop this paragraph - doc page already has a link to take folks to the "board sources" which should be enough.
(In any case bullet list should have been preceded with an empty line btw)